__tests__/frontmatter.js

```diff
@@ -53,8 +54,18 @@ function validateRuleFrontmatter({ frontmatter }, metaData) {
 	}
 	if (rule_type.toLowerCase() === `atomic`) {
 		test('has optional property `input_aspects` when `rule_type = atomic`', () => {
-			expect(frontmatter).toHaveProperty('input_aspects')
 			expect(frontmatter).not.toHaveProperty('input_rules')
+
+			expect(frontmatter).toHaveProperty('input_aspects')
+			expect(Array.isArray(frontmatter.input_aspects)).toBe(true)
+			expect(frontmatter.input_aspects.length > 0).toBe(true)
+			expect(
+				frontmatter.input_aspects.every(aspect => {
+					const isTypeString = Object.prototype.toString.call(aspect) === '[object String]'
+					const isNotMultipleAspects = /\s-\s/.test(aspect) === false // ensure it is not concatantated aspects like `"DOM Tree - CSS Styling"`
+					return isTypeString && isNotMultipleAspects
+				})
+			).toBe(true)
```

Comment:
What error messages are going to be displayed if these tests fail? Are they understandable?

Reply:
I have added custom messages, example below:

![image](https://user-images.githubusercontent.com/20978252/76857502-7d2b9480-684d-11ea-8c1f-099c9b4dcbf3.png)